changelog/unreleased/add-spaceshared-event.md

```diff
@@ -0,0 +1,7 @@
+Enhancement: Add SpaceShared event
+
+We added an event that is emmitted when somebody shares a space.
+
+https://github.com/owncloud/ocis/pull/4564
```

Comment:
@butonic @dragonchaser Please check during reviews that the first link is always the reva pull request. This changelog entry is broken, cannot be rendered because the first id doesn't resolve inside of reva